test/integration/crossPartition.spec.ts

```diff
@@ -340,6 +341,34 @@ describe("Cross Partition", function() {
       await executeQueryAndValidateResults({ query, options, expectedOrderIds: expectedOrderedIds });
     });
 
+    it("Validate DISTINCT Query", async function() {
+      // simple order by query in string format
+      const query = "SELECT DISTINCT VALUE r.spam3 FROM root r";
+      const options = {
+        enableCrossPartitionQuery: true,
+        maxItemCount: 2
+      };
+
+      const expectedOrderedIds = Array.from(new Set(documentDefinitions.map(r => r["spam3"])));
+
+      // validates the results size and order
+      await executeQueryAndValidateResults({ query, options, expectedOrderIds: expectedOrderedIds });
+    });
+
+    it("Validate DISTINCT OrderBy Query", async function() {
+      // simple order by query in string format
+      const query = "SELECT DISTINCT VALUE r.spam3 FROM root r order by r.spam3";
+      const options = {
+        enableCrossPartitionQuery: true,
+        maxItemCount: 2
```

Comment:
Please add tests that vary the `maxItemCount`. `maxItemCount` = 1 and collectionSize are special edge cases.

`maxItemCount` = 1 means the backend will essentially do no work and client is responsible for doing all the DISTINCT
`maxItemCount` = collectionSize means the backend will essential do all the work and the clients job is just not to lose any documents.

Reply:
Done